### DISCUSSION TITLE
请问可以搞一个微信群或者其他的交流群么

### DISCUSSION THREAD

**Discussion by yuhao0102** [mlc-ai/mlc-zh#11] [General]
微信群可能更方便大家讨论问题和答疑

**Comment by Hzfengsy** [comment]
我们没有官方微信交流群，github discussion将作为学生和老师、助教互动的主要途径。但我们支持各类自发组织的交流、讨论群 :)

**Reply by antonia0912** [reply]
> 我们没有官方微信交流群，github discussion将作为学生和老师、助教互动的主要途径。但我们支持各类自发组织的交流、讨论群 :)

 我建了一个机器学习编译的微信群，可以给你一个管理员权限（如果你需要） @Hzfengsy 

<img src="https://tva1.sinaimg.cn/large/e6c9d24ely1h4ml4kssw5j20s310u76v.jpg" width = "300" height = "400" alt="图片名称" align=center />

**Reply by glorytheksy** [reply]
> > 我们没有官方微信交流群，github discussion将作为学生和老师、助教互动的主要途径。但我们支持各类自发组织的交流、讨论群 :)
> 
> 我建了一个机器学习编译的微信群，可以给你一个管理员权限（如果你需要） @Hzfengsy
> 
> <img alt="图片名称" width="300" height="350" src="https://camo.githubusercontent.com/b2371b19d8f0ef68b790da8a4aea2536894ff1c321ebded8e301809a27381c8d/68747470733a2f2f747661312e73696e61696d672e636e2f6c617267652f65366339643234656c793168336674726a777362636a323073333130753076772e6a7067">

你好，可以再发下微信群吗？

**Reply by antonia0912** [reply]
> > > 我们没有官方微信交流群，github discussion将作为学生和老师、助教互动的主要途径。但我们支持各类自发组织的交流、讨论群 :)
> > 
> > 
> > 我建了一个机器学习编译的微信群，可以给你一个管理员权限（如果你需要） @Hzfengsy
> > <img alt="图片名称" width="300" height="350" src="https://camo.githubusercontent.com/b2371b19d8f0ef68b790da8a4aea2536894ff1c321ebded8e301809a27381c8d/68747470733a2f2f747661312e73696e61696d672e636e2f6c617267652f65366339643234656c793168336674726a777362636a323073333130753076772e6a7067">
> 
> 你好，可以再发下微信群吗？

已更新，请查看

**Reply by Don1110** [reply]
你好，二维码又过期了，可以再发一下吗

**Reply by HLearning** [reply]
> > 我们没有官方微信交流群，github discussion将作为学生和老师、助教互动的主要途径。但我们支持各类自发组织的交流、讨论群 :)
> 
> 我建了一个机器学习编译的微信群，可以给你一个管理员权限（如果你需要） @Hzfengsy
> 
> <img alt="图片名称" width="300" height="400" src="https://camo.githubusercontent.com/c6e2c9342bff1e571a5bac83aa64055a75da905d30c18d566e479a998c0be15c/68747470733a2f2f747661312e73696e61696d672e636e2f6c617267652f65366339643234656c793168346d6c346b737377356a323073333130753736762e6a7067">

二维码过期了， 你看我还有机会加群嘛？

**Reply by Andy1314Chen** [reply]
二维码又过期了，可以再发一下吗?

**Reply by antonia0912** [reply]
群满 200 人没办法扫码进了，加我微信 Hyperai01 备注 TVM 进群，我邀请你

**Reply by niceyangyizhi** [reply]
二维码又过期了，可以再发一下吗?

**Reply by antonia0912** [reply]
> 二维码又过期了，可以再发一下吗?

加微信  Hyperai01 吧，备注一下，我邀请你进群

**Comment by conanhujinming** [comment]
我创建了一个qq群 696249157，感觉qq群加群会方便一些，欢迎官方人员入驻~
